feat: add database-driven organizer page

Jer-Pha opened the pull request:
Resolves #64 

Organizers will need to be manually added to the database via /admin/. By default, these changes will only show active organizers and will link to their Meetup profile and LinkedIn (optional).

There are commented lines in `views.py`, `__init__.py`, and `organizers.html` that could display past organizers below active organizers. This would be a good way to honor those that have helped in the past. This would require more manual work adding extra people in /admin/ and changing their active status to False.

Jer-Pha committed:
feat: add database-driven organizer page

Jer-Pha commented:
Hmm, not sure why it failed the check.

davidfischer commented:
> Hmm, not sure why it failed the check.

There's two reasons if you clickthrough to the CI details (you can see those right?). Firstly, there was a minor code formatting issue that needs to be cleaned up with Black or with the [pre-commit](https://github.com/sandiegopython/pythonsd-django?tab=readme-ov-file#developing). However, the code has 100% test coverage and is set to fail if it drops below that. That means that if you add new code, the code needs to be executed by tests. You added a test for it, but there's a couple lines it didn't execute. Notably since no organizers were added in the test, none of those lines in the template were executed.

For now, I say we just keep the code to the active organizers. The main goal of the page is so that folks can contact an organizer if they have a question.

Jer-Pha committed:
Removed commented lines from organizers.html

Jer-Pha committed:
Removed commented line from __init__.py

Jer-Pha committed:
Removed commented lines from views.py

Jer-Pha committed:
Added model test case to __init__.py

davidfischer commented:
Looks like we're working simultaneously. I [created a branch](https://github.com/sandiegopython/pythonsd-django/compare/davidfischer/Jer-Pha-organizers?expand=1) (branched from this) that added a few things that were missing:

- There was no database migration
- I fixed up the tests/linting
- I removed the inactive organizers code

Jer-Pha committed:
Moved test_organizers() from TestBasicViews to OrganizerTestCase

Jer-Pha commented:
> Looks like we're working simultaneously. I [created a branch](https://github.com/sandiegopython/pythonsd-django/compare/davidfischer/Jer-Pha-organizers?expand=1) (branched from this) that added a few things that were missing:
> 
> * There was no database migration
> * I fixed up the tests/linting
> * I removed the inactive organizers code

I'll hold off on any other changes, sorry about that!

davidfischer commented:
> I'll hold off on any other changes, sorry about that!

Go ahead but you might start from or at least look at the other branch (we can merge it into this PR too). One thing to think about is should there be an image for organizers? Should it just be the image from meetup? How do we get that image?

Edit: some discussion of this topic [here](https://github.com/sandiegopython/pythonsd-django/issues/64#issuecomment-2138658651).
